Library/ViewModels/CommentsViewModel.swift

```diff
@@ -252,10 +260,21 @@ public final class CommentsViewModel: CommentsViewModelType,
       self.loadCommentsAndProjectIntoDataSource.ignoreValues()
     )
 
-    self.configureFooterViewWithState = Signal.merge(
+    let hideFooterView: Signal<CommentTableViewFooterViewState, Never> = Signal.merge(
       initialLoadOrReload.mapConst(.hidden),
+      errors
+        .mapConst(.hidden)
+        .withLatestFrom(hasRequestedNextPage.filter(isFalse))
+        .map(first)
```

Comment:
Does this mean we only hide the footerview if it's a first page request with an error?

Reply:
Add some inline docs here.

Reply:
Yes that is what it means @msadoon I will add some inline docs